test/gpu_reference_kernel.cpp

```diff
@@ -447,6 +482,13 @@ struct gpu_reference_conv_2d : gpu_reference_kernel_base
                                  wei.data.data(),
                                  sizeof(TRef) * wei_sz,
                                  hipMemcpyHostToDevice) == hipSuccess);
+                // TODO(Amber): copy output before computation because output may
+                // be not be packed, and convolution may update only a subset of
+                // indices
+                EXPECT(hipMemcpy(out_dev,
```

Comment:
hipMemcpy should not be used with OpenCL backend.

BTW We no longer support OCL backend so you can simply keep this as is or remove the whole OpenCL block of code (I would prefer the former due to [this](url))

Reply:
Sorry, the hyperlink to `this` above is broken. I don't think I have any `hipMemCpy` outside of `#elif MIOPEN_BACKEND_HIP`, so OpenCL backend is unaffected. I could perhaps add the corresponding `clEnqueWriteBuffer` to OpenCL backend if we care to maintain it. If not, I prefer deleting all `OpenCL` code from this file.

Reply:
@amberhassaan URL fixed, sorry.

Reply:
Discussed this with Jehandad and we decided that it's OK to remove OpenCL code. I can do that in this PR @atamazov if you're OK with that or leave it as a follow up action.

Reply:
@amberhassaan Removing looks good -- as I said, the final decision is up to you.

Reply:
Will remove (next commit once CI completes).